cmd/bpf2go/internal/module.go

```diff
@@ -0,0 +1,14 @@
+package internal
+
+import "runtime/debug"
+
+// CurrentModule returns the module containing the main package.
+func CurrentModule() string {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		// Fall back to constant since bazel doesn't support BuildInfo.
+		return "github.com/cilium/ebpf"
+	}
+
+	return bi.Main.Path
```

Comment:
https://pkg.go.dev/runtime/debug#BuildInfo

This means that any outside program trying to use the code template will insert its own package instead of `github.com/cilium/ebpf`.
So even though the template functions are now public, it is not possible to use them outside of this repo and generate functional bindings.

Reply:
Ugh, good point. I guess removing the parameter outright it is.

Reply:
Testing this properly is a headache as well.

Reply:
Goodbye BuildInfo, it was probably a bad idea from the start.